temporal-sdk/src/main/java/io/temporal/nexus/WorkflowClientOperationHandlers.java

```diff
@@ -52,9 +36,8 @@ public static <T, R> OperationHandler<T, R> sync(
   public static <T, R> OperationHandler<T, R> fromWorkflowMethod(
       WorkflowMethodFactory<T, R> startMethod) {
     return new RunWorkflowOperation<>(
```

Comment:
Doesn't have to be in this PR but I would to call this `WorkflowRunOperation` across all SDKs.